### PR TITLE
Expands Surface Security among other things

### PR DESCRIPTION
**Pull request by Very-Soft**
In continuing with the plan - https://github.com/VOREStation/VOREStation/issues/6122 :

Expands Surface Security
Nerfs Space Security
Moves Public Teleporter to Z3
Implements Rykka's Robotics changes ( https://github.com/VOREStation/VOREStation/pull/9194 )

z3
z2
z1

### CODE REVIEW

**Comment by KasparoVy** [comment]
I guess this means great cargonia will be landside and mining will once again have to launch from the planet to the stars?

(Totally fine with that! Just wanted to confirm future plans)

**Comment by Very-Soft** [comment]
Negative, Cargonia is a job that does revolve around a shuttle, it will remain in space (though it does have a cargo shop on the surface now)

**Comment by MrSign** [comment]
Noicely done, glad to see you're continuing to work on it! Greatly approve of the changes!

**Comment by Andromeda-K22** [comment]
I think the three level security setup is a little bit confusing. The current brig is a bit of a sprawl, but it is clearly deliniated into A. the functions security, and B. perma/cells. 

You've kinda spread the cells out all over the place. Two on the first level, and then the rest of the brig on the third level. (from the top down) I'd suggest putting all the cells together instead of having them spread out, or combine those two cells into an easily-accessible 'drunk tank'.

**Comment by Very-Soft** [comment]
> I think the three level security setup is a little bit confusing. The current brig is a bit of a sprawl, but it is clearly deliniated into A. the functions security, and B. perma/cells.
> 
> You've kinda spread the cells out all over the place. Two on the first level, and then the rest of the brig on the third level. (from the top down) I'd suggest putting all the cells together instead of having them spread out, or combine those two cells into an easily-accessible 'drunk tank'.

My idea for this was simply to allow for more proper brig functionality to the idea of the drunk tank without putting it into that stigma, while also retaining the closeness to where security folk are likely to be hanging out. 

You have 2 cells near the front, right next to Processing and IAA and in view of the hallway, intended primarily for short sentences. It's easy to access, relatively easy to break into and out of with the right tools/help. These should be easy for security to use for RP or for basic low priority every day crimes without having to go all the way down to perma. 

Where the 2 cells in perma are far away and a little harder to escape without help. These are meant for high security prisoners, further away and with a little more pomp to them, you're probably only going to go here if you really fucked up.

Basically the way the space brig is, is way too big for what we handle in addition to being so far away from everything that if you get locked up you won't see or interact with anyone until your sentence is up. It's far away and requires help to get out of.

The existing drunk tank is mostly okay on the other end, but there's only one timer and no way to lock up anything sensitive the person might be carrying, so. I thought it might be better if there were 2 small holding cells closer to the front desk, to make things easier on sec and less boring for small time criminals.
